src/amp/abTests.ts

```diff
@@ -0,0 +1,16 @@
+import {
+    ExperimentFullConfig
```

Comment:
Importing a type from an AMP component is confusing.

Reply:
Moved Types out of this file.